pandas/util/testing.py

```diff
@@ -853,12 +858,25 @@ def _get_ilevel_values(index, level):
     if check_categorical:
         if is_categorical_dtype(left) or is_categorical_dtype(right):
             assert_categorical_equal(left.values, right.values,
-                                     obj='{obj} category'.format(obj=obj))
+                                     obj=map_obj('{obj} category', obj))
+
+
+def join_obj(obj):
+    obj = com._maybe_make_list(obj)
+    return ' and '.join(obj)
+
+
+def map_obj(form, objs, **kwargs):
+    assert isinstance(form, str)
```

Comment:
don't put these here, rather in testing.pyx. I also don't particularly like these names.